src/python/workshop/utilities.py

```diff
@@ -38,8 +38,6 @@ async def get_file(self, project_client: AIProjectClient, file_id: str, attachme
         attachment_part = attachment_name.split(":")[-1]
         file_name = Path(attachment_part).stem
         file_extension = Path(attachment_part).suffix
-        if not file_extension:
```

Comment:
This is needed as occasionally the CI wont return the file with the file extension and then it creates a bad user experience for the attendee

Reply:
reverted this change back